### PR TITLE
bpo-34134: Advise to use imap or imap_unordered when handling long iterables.

### PR DESCRIPTION
**Pull request by Windsooon**
https://bugs.python.org/issue34134

### CODE REVIEW

**Comment by pablogsal** [comment]
@Windsooon Could you fix the trailing whitespace?

```
python3 tools/rstlint.py -i tools -i ./venv -i README.rst
[2] library/multiprocessing.rst:2145: default role used
[1] library/multiprocessing.rst:2144: trailing whitespace
[1] library/multiprocessing.rst:2145: trailing whitespace
2 problems with severity 1 found.
1 problem with severity 2 found.
```

**Comment by Windsooon** [comment]
@pablogsal Thank you for review. I already fix it. And I also find another problem about imap, and I added in my commit.

File: Doc/library/multiprocessing.rst [hunk 1]
```diff
@@ -2141,6 +2141,10 @@ with the :class:`Pool` class.
       the process pool as separate tasks.  The (approximate) size of these
       chunks can be specified by setting *chunksize* to a positive integer.
 
+      Note that it may cause high memory usage for very long iterables. Consider
+      using :func:`imap` or :func:`imap_unordered` with explicit *chunksize*
```
Comment (pitrou) [inline Doc/library/multiprocessing.rst]: The right markup here would probably be `:meth:`, not `:func:`.

**Review by BoboTiG — APPROVED** [review]
Thank you! Changes look good to me.

**Comment by Windsooon** [comment]
This PR should be merged, right?

**Comment by pitrou** [comment]
@Windsooon yes sorry, I had totally overlooked that you had done the necessary changes. Sorry for the wait.

**Comment by Windsooon** [comment]
@pitrou Thank you and have a nice weekend.

**Pull request merged**